sc2reader/scripts/sc2json.py

```diff
@@ -15,28 +16,31 @@ def main():
         default=None,
         help="The per-line indent to use when printing a human readable json string",
     )
-    parser.add_argument(
-        "--encoding",
-        "-e",
-        type=str,
-        default="UTF-8",
-        help="The character encoding use..",
-    )
     parser.add_argument(
         "path",
         metavar="path",
         type=str,
         nargs=1,
         help="Path to the replay to serialize.",
     )
+    if sys.version_info.major < 3:
+        parser.add_argument(
+            "--encoding",
+            "-e",
+            type=str,
+            default="UTF-8",
+            help="The character encoding use..",
+        )
+
     args = parser.parse_args()
 
     factory = sc2reader.factories.SC2Factory()
-    try:
```

Comment:
This change above is good.
The change below is not really required because the try / except block does the same thing.

Reply:
If I would leave the original try/except it would throw an AttributeError before the TypeError though if args.encoding doesn't exist